base-notebook/start.sh

```diff
@@ -108,9 +109,9 @@ if [ "$(id -u)" == 0 ] ; then
     # directory if it doesn't already exist, and update the current working
     # directory to the new location if needed.
     if [[ "${NB_USER}" != "jovyan" ]]; then
-        if [[ ! -e "/home/${NB_USER}" ]]; then
+        if [[ ! -e "/home/ogiorgis"  ||  "${COPY_HOME}" == "yes" || "${COPY_HOME}" == "1" ]]; then
```

Comment:
`ogiorgis` was probably the name of your user and it should be changed.
And there are too many spaces near `||`.